commit 092ff9eefe6d8946f747a94506a2e7c53ee286e8
Author:     Tizen Build Automation <tizenbuild@samsung.com>
AuthorDate: Wed Oct 28 03:30:52 2026 +0000
Commit:     Tizen Build Automation <tizenbuild@samsung.com>
CommitDate: Wed Oct 28 03:30:52 2026 +0000

    Update candidate_snapshot : TIZEN/Tizen/Tizen-Base-Toolchain/project_meta,TIZEN/Tizen/Tizen-Base-Dev/project_meta,
---
 TIZEN/Tizen/Tizen-Base-Dev/candidate_snapshot      | 22 +++++++++++-----
 .../Tizen/Tizen-Base-Toolchain/candidate_snapshot  | 30 ++++++++++++----------
 2 files changed, 32 insertions(+), 20 deletions(-)

diff --git a/TIZEN/Tizen/Tizen-Base-Dev/candidate_snapshot b/TIZEN/Tizen/Tizen-Base-Dev/candidate_snapshot
index 70abe4a..31f0769 100644
--- a/TIZEN/Tizen/Tizen-Base-Dev/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Base-Dev/candidate_snapshot
@@ -1,9 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <snapshotinfo>
-  <candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/" />
-  <candidate target="tizen-base-dev_20260930.023443" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/" />
-  <candidate target="tizen-base-dev_20261013.023909" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/" />
-  <base_candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/" />
-  <base_candidate target="tizen-base-dev_20260930.023443" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/" />
-  <base_candidate target="tizen-base-dev_20261013.023909" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Dev/" />
+  <candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <candidate target="tizen-base_20260924.053151" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <candidate target="tizen-base_20261006.004420" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <candidate target="tizen-base_20261019.064604" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <candidate target="tizen-base_20261021.010449" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <candidate target="tizen-base_20261022.051655" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <candidate target="tizen-base_20261023.004035" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <candidate target="tizen-base_20261026.005418" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <base_candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <base_candidate target="tizen-base_20260924.053151" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <base_candidate target="tizen-base_20261006.004420" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <base_candidate target="tizen-base_20261019.064604" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <base_candidate target="tizen-base_20261021.010449" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <base_candidate target="tizen-base_20261022.051655" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <base_candidate target="tizen-base_20261023.004035" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <base_candidate target="tizen-base_20261026.005418" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
 </snapshotinfo>
\ No newline at end of file
diff --git a/TIZEN/Tizen/Tizen-Base-Toolchain/candidate_snapshot b/TIZEN/Tizen/Tizen-Base-Toolchain/candidate_snapshot
index 5845695..31f0769 100644
--- a/TIZEN/Tizen/Tizen-Base-Toolchain/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Base-Toolchain/candidate_snapshot
@@ -1,17 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <snapshotinfo>
-  <candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <candidate target="tizen-base-toolchain_20260924.094908" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <candidate target="tizen-base-toolchain_20261013.024319" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <candidate target="tizen-base-toolchain_20261013.140248" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <candidate target="tizen-base-toolchain_20261020.074527" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <candidate target="tizen-base-toolchain_20261026.051209" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <candidate target="tizen-base-toolchain_20261026.051553" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <base_candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <base_candidate target="tizen-base-toolchain_20260924.094908" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <base_candidate target="tizen-base-toolchain_20261013.024319" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <base_candidate target="tizen-base-toolchain_20261013.140248" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <base_candidate target="tizen-base-toolchain_20261020.074527" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <base_candidate target="tizen-base-toolchain_20261026.051209" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <base_candidate target="tizen-base-toolchain_20261026.051553" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
+  <candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <candidate target="tizen-base_20260924.053151" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <candidate target="tizen-base_20261006.004420" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <candidate target="tizen-base_20261019.064604" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <candidate target="tizen-base_20261021.010449" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <candidate target="tizen-base_20261022.051655" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <candidate target="tizen-base_20261023.004035" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <candidate target="tizen-base_20261026.005418" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <base_candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <base_candidate target="tizen-base_20260924.053151" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <base_candidate target="tizen-base_20261006.004420" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <base_candidate target="tizen-base_20261019.064604" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <base_candidate target="tizen-base_20261021.010449" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <base_candidate target="tizen-base_20261022.051655" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <base_candidate target="tizen-base_20261023.004035" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
+  <base_candidate target="tizen-base_20261026.005418" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/" />
 </snapshotinfo>
\ No newline at end of file
